tsocial/nomad-proxy
# TLS communication for nomad client proxy.

Current requirement: Clicks deployment.
testdata: contains dummy certs for server and client.
Testing done as follows:
1. Start a nomad agent in dev mode (server runs on server certs) which demands a certificate.
2. Start a nomad-client-proxy with the client certs and `server-addr` = nomad server's url. The client should be able to speak to the server.
